modules/social_features/social_group/src/EventSubscriber/RedirectSubscriber.php

```diff
@@ -39,13 +39,18 @@ public function checkForRedirection(RequestEvent $event) {
     }
 
     // Get the current route name for the checks being performed below.
-    $routeMatch = \Drupal::routeMatch()->getRouteName();
+    $route_name = \Drupal::routeMatch()->getRouteName();
 
     // Redirect the group content collection index to the group canonical URL.
-    if ($routeMatch === 'entity.group_content.collection') {
+    if ($route_name === 'entity.group_content.collection') {
       $event->setResponse(new RedirectResponse(Url::fromRoute('entity.group.canonical', ['group' => $group->id()])
         ->toString()));
     }
+    elseif ($route_name === 'view.group_pending_members.page_1') {
```

Comment:
@ksis Let's add an explanation of why we add this.

Reply:
Hi @Ressinel. I added explanation. Please, check.